.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+
+updates:
+  - package-ecosystem: 'cargo'
+    directory: '/local-registry'
+    schedule:
+      interval: 'weekly'
+    groups:
+      - all:
+        patterns:
+         - "*"
+    versioning-strategy: 'increase'
```

Comment:
`/local-registry` is likely going to be detected as a library by dependabot, in which case it doesn't update as aggressively as for applications. This forces the most eager update strategy.